lib/mayaUsd/ufe/UsdShaderNodeDef.cpp

```diff
@@ -435,6 +436,12 @@ Ufe::NodeDefs UsdShaderNodeDef::definitions(const std::string& category)
 
         result.reserve(shaderNodeDefs.size());
         for (const SdrShaderNodeConstPtr& shaderNodeDef : shaderNodeDefs) {
+            // Skip deprecated USD native shaders that have "empty.glslfx" as their resolved URI.
+            if (!shaderNodeDef->GetResolvedImplementationURI().empty()
+                && TfStringEndsWith(
+                    shaderNodeDef->GetResolvedImplementationURI(), "empty.glslfx")) {
+                continue;
+            }
```

Comment:
See [usd/usdHydra/shaders/shaderDefs.usda](https://github.com/PixarAnimationStudios/OpenUSD/blob/100f1f926763453bf258a29c9b98ae70dcb87d17/pxr/usd/usdHydra/shaders/shaderDefs.usda#L15) for details.